### PR TITLE
Fix dixmaan name

### PR DESCRIPTION
**Pull request by d-monnet**

### CODE REVIEW

File: src/ADNLPProblems/dixmaan_efgh.jl [hunk 1]
```diff
@@ -21,14 +21,14 @@ function dixmaane(;
            sum(T(i / n) * T(δ) * x[i] * x[i + 2 * m] for i = 1:m)
   end
   x0 = 2 * ones(T, n)
-  return ADNLPModels.ADNLPModel(f, x0, name = "dixmaan_efgh"; kwargs...)
+  return ADNLPModels.ADNLPModel(f, x0, name = "dixmaan_e"; kwargs...)
```
Comment (dpo) [inline src/ADNLPProblems/dixmaan_efgh.jl:24]: Thanks. I don't think there should be an underscore though.

**Review by dpo — CHANGES_REQUESTED** [review]

File: src/ADNLPProblems/dixmaan_efgh.jl (hunk 1 above)
Comment (tmigot) [reply src/ADNLPProblems/dixmaan_efgh.jl:24]: In the `meta` https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl/blob/1ef559df2d101557f23e7eed8a2bff09c76be05a/src/Meta/dixmaane.jl#L7
there is no underscore.

Comment (d-monnet) [reply src/ADNLPProblems/dixmaan_efgh.jl]: I'll make a new pull request with no underscore

**Comment by d-monnet** [comment]
New pull request with no underscore in the dixmaan problems name will be done

**Pull request closed without merging**